### ISSUE TITLE
Add HuberRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#69]
Add HuberRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference HuberRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/HuberRegressor.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.HuberRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an HuberRegressor as a potential best fit model.

**Comment by hirun-rashmitha** [comment]
I can go through this issue. I will make a commit with changes!

**Comment by hirun-rashmitha** [comment]
I will go through this. I will make a commit ASAP
